README.md

```diff
@@ -7,9 +7,9 @@ Modify the iOS Simulator so that it has a perfect status bar, then run your app
 * 5 bars of cellular signal and full WiFi bars are displayed.
 * Tue Jan 9 is displayed for the date (iPad only)
 
-## Xcode 11 beta 4 and `simctl status_bar`! 🚀
+## Can't I just use `xcrun simctl status_bar`? 🚀
 
-Xcode 11 beta 4 includes support for perfect status bars without SimulatorStatusMagic! 🎉 Run `xcrun simctl status_bar` with beta 4 or later installed and rejoice! This project will be going away soon, which is great news.
+Starting in Xcode 11, the `simctl` command line tool includes a `status_bar` option that allows you to override the appearance of the status bar in the simulator. Hopefully this will eventually supercede the need for SimulatorStatusMagic, but at the moment it still has holes that make this project continue to be relevant. In particular, `simctl status_bar` does not currently provide a way to add localized date and time strings in the status bar.
```

Comment:
This is great 👍

You can just hear my optimism from the option being new, can't you!